sale_stock_security/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Sale Stock Security",
+    "summary": """
+        Now users belonging to the group ‘User: All documents’ will only see orders from their warehouse.
+    """,
+    "author": "Solvos",
+    "license": "LGPL-3",
+    "version": "17.0.1.0.0",
+    'category': "Operations/Sale",
+    "website": "https://github.com/solvosci/slv-sale",
+    "depends": ["sale_stock", "sales_team"],
```

Comment:
`sales_team` is added through `sale`, that already is added by `sale_stock`. Then, we can remove `sales_team` dependency, is actually included.